Configuration/odin-views.json

```diff
@@ -5,6 +5,7 @@
             "configuration": {
                 "": "odin/static",
                 "api/": "odin/api",
+                "api/delete/": "odin/delete",
```

Comment:
Why is this done here? Aren't there user APIs somewhere else?

Reply:
Just copy solution from similar other APIs. Yes it can be used in `api/`.